code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -426,6 +426,24 @@ GLOBAL_LIST_EMPTY(dynamic_forced_rulesets)
 	generate_budgets()
 	set_cooldowns()
 	log_dynamic("Dynamic Mode initialized with a Threat Level of... [threat_level]! ([round_start_budget] round start budget)")
+	SSblackbox.record_feedback(
+		"associative",
+		"dynamic_threat",
+		1,
+		list(
+			"server_name" = CONFIG_GET(string/servername),
```

Comment:
can this be `SERVERSQLNAME` instead? It'll be easier to filter for/on.

Reply:
Not gonna lie, I don't know what the difference is in practice, but can do if it helps!